pkg/storage/fs/lister.go

```diff
@@ -15,7 +16,12 @@ func (l *storageImpl) List(ctx context.Context, module string) ([]string, error)
 	loc := l.moduleLocation(module)
 	fileInfos, err := afero.ReadDir(l.filesystem, loc)
 	if err != nil {
-		return nil, errors.E(op, err)
+		kind := errors.KindUnexpected
+		if _, ok := err.(*os.PathError); ok {
```

Comment:
would ~`os.IsNotExist` work here?

Reply:
yes, it can, but at this point, i wanted to check errors from Open specifically which are wrapped into PathError. 
What IsNotExist does is that it compares error msg of the error. But i'm ok with isNotExist if you prefer this way

Reply:
@michalpristas as long as PathError _always_ means NotFound, I'm cool with either one. Thanks!